test/SwooleStreamTest.php

```diff
@@ -26,7 +26,7 @@ public function setUp()
         }
         $this->request = $this->prophesize(SwooleHttpRequest::class);
         $this->request
-            ->rawcontent()
+            ->rawContent()
```

Comment:
Is this true across all Swoole versions? We grabbed this name directly from source at one point, which is why I'm asking...

Reply:
https://github.com/swoole/swoole-src/commit/b8b4c3917b4a9bc61851a9547506d3bc444b6905#diff-3ced702c2439aa81b4faccece28f4c90

It's looks no changelog in [swoole@v4.3.0](https://github.com/swoole/swoole-src/releases/tag/v4.3.0).

C extension can be ignore case，when calls the `$swooleRequest->rawContent()`.

Reply:
@weierophinney Method name is not case sensitive, `rawContent()` would be a better name.